test/test_dos.py

```diff
@@ -6285,6 +6285,7 @@ def setUpClass(cls):
             "test_mfs_sfn_truename": KNOWNFAIL,
             "test_floppy_vfs": KNOWNFAIL,
             "test_pcmos_build": KNOWNFAIL,
+            "test_libi86_build": KNOWNFAIL,
```

Comment:
What is the problem?

Reply:
Some tests stalling and producing no output, see https://github.com/tkchia/libi86/issues/5. Those tests use dosemu's  `-input` option which seems to be a common factor, though I haven't started debugging it myself. It seems curious that the same tests work okay with FDPP and MS-DOS.

Reply:
That issue is now fixed in libi86